src/lib/frontend/models.ml

```diff
@@ -481,6 +481,11 @@ let pp_constant ppf (_, t) =
   Format.fprintf ppf "%a" SmtlibCounterExample.pp_dummy_value_of_type t
 
 let output_concrete_model fmt props ~functions ~constants ~arrays =
+  if ModelMap.(is_suspicious functions || is_suspicious constants
+               || is_suspicious arrays) then
+    Format.fprintf fmt "; This model is a best-effort. It includes symbols
+        for which model generation is known to be incomplete. @.";
```

Comment:
I don't think we should be flushing the printer here, `@,` would be preferrable, but I am not sure we are in a box here, so let's leave it that way for now.